src/Module.Server/ModuleData/native_parameters.xml

```diff
@@ -14,5 +14,8 @@
     <native_parameter id="running_backward_speed_multiplier" value="0.65" />  <!-- Native = 0.5 -->
     <native_parameter id="running_sides_speed_multiplier" value="0.775" />   <!-- Native = 0.65-->
     <native_parameter id="ladder_climb_up_speed_multiplier"  value="0.88" /> <!-- Native = 0.68-->
+    <native_parameter id="humanoid_physics_capsule_radius_multiplier_for_friend" value="1.3" /> <!--Native = 0.95-->
+    <native_parameter	id="humanoid_physics_capsule_radius_multiplier_for_enemy"	value="1.8" /> <!--Native = 1.5-->

```

Comment:
Do 1.6 for an equivalent end capsule of 0.29.

Reply:
Thanks for the response, I believe 1.6 is not enough in the short term, with ranged shooting past shields and people getting too close into each other causing more glancing?
But yes 1.6 will help a bit towards this

Reply:
please start with 1.6